sync/shared/variables.yaml

```diff
@@ -14,7 +14,10 @@ windows_node_ip: "10.20.30.11"
 # from https://apt.kubernetes.io/
 k8s_linux_registry: "gcr.io/k8s-staging-ci-images"
 k8s_linux_kubelet_deb:  "1.21.0"
-k8s_linux_apiserver: "ci/v1.22.0-alpha.3.31+a3abd06ad53b2f"
+
+# If no version provided for k8s_linux_apiserver it will take
+# the latest ci image version. Example: ci/v1.22.6-rc.0.55+487eef699dc8ad
```

Comment:
Should we document this in the README?

Reply:
In general, yes, we can improve README. The idea with this patch is don't change the existing behavior just auto detect the last tagged version for ci.

Reply:
I meant, generally, we can mention all fields in the README.md from variables and also mention that we have the tiny version.

Reply:
```
$ ls sync/shared/va*
sync/shared/variables-lite.yaml 
sync/shared/variables.yaml
```

Reply:
```
# cat variables.yaml | grep ": " | egrep -v ^\#
kubernetes_sha: "286709307da5adb62e78d989b7ed846355d5c88e"
kubernetes_compatibility:  "1.21.2"
overwrite_windows_bins: "false"
kubelet_path:  "./sync/windows/bin/kubelet.exe"
kubeproxy_path:  "./sync/windows/bin/kube-proxy.exe"
windows_node_ip: "10.20.30.11"
k8s_linux_registry: "gcr.io/k8s-staging-ci-images"
k8s_linux_kubelet_deb:  "1.21.0"
k8s_linux_apiserver: "ci/v1.22.0-alpha.3.31+a3abd06ad53b2f"
k8s_linux_kubelet_nodeip: "10.20.30.10"
overwrite_linux_bins: false
linux_ram: 4096
linux_cpus: 4
windows_ram: 6048
windows_cpus: 4
cni: "antrea"
```

16 entries in variables file that probably good to document.
```
# cat variables.yaml | grep ": " | egrep -v ^\# | wc -l
16
```

Reply:
OK, we can do this in a separate PR